File: agent/src/main/java/io/scalac/agent/AkkaPersistenceAgentState.java
package io.scalac.agent;

import java.util.Map;
import java.util.concurrent.ConcurrentHashMap;

public class AkkaPersistenceAgentState {

    public static Map<String, Long> recoveryStarted = new ConcurrentHashMap<>(100);
    public static Map<String, Long> recoveryMeasurements = new ConcurrentHashMap<>(100);
}

File: agent/src/main/scala/io/scalac/agent/AkkaPersistenceAgentState.scala
package io.scalac.agent

import java.util.concurrent.ConcurrentHashMap

object AkkaPersistenceAgentState {
  val recoveryStarted: java.util.Map[String, Long]      = new ConcurrentHashMap[String, Long](100)
  val recoveryMeasurements: java.util.Map[String, Long] = new ConcurrentHashMap[String, Long](100)
}
